inscriptions/ISic001022.xml

```diff
@@ -193,17 +198,20 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140501">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AL">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="omitted">Φ</supplied>η<supplied reason="omitted">λι</supplied>κίσι-<lb n="2"/>μος
-                    χ<supplied reason="omitted">ρ</supplied>η-<lb n="3"/>στὸς καὶ ἄ-<lb n="4"/>μεμπτος, ἔζησεν ἔτη
-                    <supplied reason="lost">—ʹ</supplied>
-                    <lb n="5"/>
+                    <lb n="1"/><persName type="attested"><name type="idionym">Φηλικίσι
```

Comment:
For future reference, we've agreed to drop 'idionym' as not adding enough useful information and generally creating more problems than it solves.